alpha/apps/kaltura/lib/request/kSessionBase.class.php

```diff
@@ -338,6 +338,37 @@ static public function getSecretsFromCache($partnerId)
 		}
 		return null;
 	}
+	
+	static public function deleteSecretCacheKey($partnerId)
+	{
+		$cacheSections = kCacheManager::getCacheSectionNames(kCacheManager::CACHE_TYPE_PARTNER_SECRETS);
+		
+		if(!$cacheSections)
+		{
+			return null;
+		}
+		$cacheKey = self::getSecretsCacheKey($partnerId);
+		KalturaLog::debug("Deleting key [$cacheKey] now [" . date('Y-m-d H:i:s', time()) . "]");
+		$deleted = false;
```

Comment:
I don't see any real meaning to this flag, if you were able to delete the value only in one of the stores why should it be true?
also you don't really check the return value of delete so you don't really know if it succeeded or failed
in my opinion you can remove the flag, you also don't use its return value

Reply:
you are correct.

Reply:
should we use the return statements and issue an error in case we were not able to delete in a store?